modules/node-shared/src/test/scala/org/tessellation/node/shared/domain/seedlist/MergeRangesSuite.scala

```diff
@@ -0,0 +1,36 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.syntax.all._
+
+import org.tessellation.node.shared.domain.seedlist.Range.mergeOverlaps
+
+import eu.timepit.refined.auto._
+import weaver.SimpleIOSuite
+import weaver.scalacheck.Checkers
+
+object MergeRangesSuite extends SimpleIOSuite with Checkers {
+
+  test("merge overlapping ranges") {
+    val actual = Range
+      .from(0L, 20L)
+      .flatMap { a =>
+        Range.from(20L, 50L).flatMap { b =>
+          Range.from(60L, 69L).flatMap { c =>
+            Range.from(62L, 65L).flatMap { d =>
+              Range.from(70L, 79L).map(List(a, b, c, d, _))
+            }
+          }
+        }
+      }
+      .map(mergeOverlaps)
+
+    val expected = Range.from(0L, 50L).flatMap { a =>
+      Range.from(60L, 69L).flatMap { b =>
+        Range.from(70L, 79L).map(List(a, b, _))
+      }
+    }
```

Comment:
Changed to

```scala
  test("merge overlapping ranges") {
    val actual = (
      Range.make(0L, 20L),
      Range.make(20L, 50L),
      Range.make(60L, 69L),
      Range.make(62L, 65L),
      Range.make(70L, 79L)
    )
      .mapN(List(_, _, _, _, _))
      .map(mergeOverlaps)

    val expected = (
      Range.make(0L, 50L),
      Range.make(60L, 69L),
      Range.make(70L, 79L)
    )
      .mapN(List(_, _, _))

    expect.same(expected, actual).pure
  }
```

Reply:
@cngo-github Since your range is inclusive, shouldn't the expected result be this?
```
    val expected = (
      Range.make(0L, 50L),
      Range.make(60L, 79L)
    )
```

Reply:
@sdavidp , the range is now inclusive.